### PR TITLE
Add tests for SkipIfEmpty funcs

### PR DESCRIPTION
**Pull request by sebrandon1**
New coverage level: `coverage: 90.9% of statements`.

To be honest these two functions seem very similar and there might be room to merge the two in a subsequent PR.

### CODE REVIEW

File: pkg/testhelper/testhelper_test.go [hunk 1]
```diff
@@ -37,3 +38,47 @@ func TestResultToString(t *testing.T) {
...
+		{ // Test Case #1 - Skip because objects is empty, no panic because []string type
+			objects: []string{},
+			skipped: true,
+		},
+		{ // Test Case #2 - Skip because objects is empty, no panic because map type
+			objects: make(map[string]string),
+			skipped: true,
+		},
+		{ // Test Case #3 - No skip because objects is populated, no panic because []string type
+			objects: []string{"test"},
```
Comment (edcdavid) [inline pkg/testhelper/testhelper_test.go:56]: add a TCs with multiple objects

Comment (sebrandon1) [reply pkg/testhelper/testhelper_test.go:56]: Done.

Comment (edcdavid) [reply pkg/testhelper/testhelper_test.go:56]: Example single test for multiple objects (just to explain what I meant):
func TestSkipIfEmptyFuncs(t *testing.T) {
	testCases := []struct {
		map1, slice1 interface{}
		skippedAll, skippedAny bool
	}{
		{ // Test Case #4 - Multiple objects
			map1: []string{"test1", "test2"},
			slice1: make(map[string]string),
			skippedAny: true,
			skippedAll: false,
		},
		// Note: Cannot test calls to panic
	}

	for _, tc := range testCases {
		result := false

		SkipIfEmptyAll(func(s string, i ...int) {
			if strings.Contains(s, "Test skipped") {
				result = true
			} else {
				result = false
			}
		}, tc.map1, tc.slice1)
		assert.Equal(t, tc.skippedAll, result)

		SkipIfEmptyAny(func(s string, i ...int) {
			if strings.Contains(s, "Test skipped") {
				result = true
			} else {
				result = false
			}
		}, tc.map1, tc.slice1)

		assert.Equal(t, tc.skippedAny, result)
	}
}

Comment (sebrandon1) [reply pkg/testhelper/testhelper_test.go:56]: Oh I see what you are saying.  I'll update!

**Review by edcdavid — APPROVED** [review]

**Review by jmontesi — APPROVED** [review]

**Pull request merged**